### PR TITLE
Issue #SB-6814 Feat : Urdu Support for Assessment Plugins

### PR DESCRIPTION
**Pull request by sivashanmugam-funtoot**
https://project-sunbird.atlassian.net/browse/SB-6814

### CODE REVIEW

File: org.ekstep.questionunit-1.0/renderer/plugin.js [hunk 1]
```diff
@@ -1,6 +1,16 @@
 org.ekstep.contentrenderer.questionUnitPlugin = Plugin.extend({
   _type: 'org.ekstep.questionUnitPlugin',
   _render: true,
+  _addUrduSupport: function(){
+    // Add to Question Title
+    var urduSupportClass = 'urdu-right-align'
+    $('.question-title-text-common-questionunit-plugins').addClass(urduSupportClass);
```
Comment (vinukumar-vs) [inline org.ekstep.questionunit-1.0/renderer/plugin.js]: Make it a small name. It is too long.
question-title-text-common-questionunit-plugins => common-qs-title

File: org.ekstep.questionunit-1.0/renderer/plugin.js [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
+
+    // Add to Question Options
+    _.each($('.option-text-common-questionunit-plugins'), function(option){
+      $(option).addClass(urduSupportClass);
```
Comment (vinukumar-vs) [inline org.ekstep.questionunit-1.0/renderer/plugin.js]: Verify for null/undefined check here before applying addClass.
$(option)

File: org.ekstep.questionunit-1.0/renderer/plugin.js [hunk 3]
```diff
@@ -32,16 +42,18 @@ org.ekstep.contentrenderer.questionUnitPlugin = Plugin.extend({
...
     var questionsetInstance = event.target;
+    var qData = JSON.parse(questionsetInstance._currentQuestion.data)
     $(questionsetInstance._constants.qsElement).html(template({
       question: this._question
     }));
 
     this.postQuestionShow(event);
-
     this.renderMath(event);
+    if(qData.question.urdu){
```
Comment (vinukumar-vs) [inline org.ekstep.questionunit-1.0/renderer/plugin.js]: If it V1 question, will this work? qData  will be undefined for V1 question right?

File: org.ekstep.questionunit.mcq-1.1/renderer/template/mcq-layouts.js [hunk 4]
```diff
@@ -462,7 +462,7 @@ MCQController.grid2.getOption = function (option, key) {
       src=<%=MCQController.pluginInstance.getAssetUrl(option.image) %> />\
   <%}%>\
   <%if(!option.image && option.text){%>\
-    <div class='mcq2-2-option-text'><%= option.text %></div>\
+    <div class='mcq2-2-option-text option-text-common-questionunit-plugins'><%= option.text %></div>\
```
Comment (vinukumar-vs) [inline org.ekstep.questionunit.mcq-1.1/renderer/template/mcq-layouts.js]: Make it class name small option-text-common-questionunit-plugins => common-option-text

File: org.ekstep.questionunit-1.0/renderer/plugin.js (hunk 3 above)
Comment (sivashanmugam-funtoot) [reply org.ekstep.questionunit-1.0/renderer/plugin.js]: Yes,
For v1 question the urdu property will not exist inside question object of qData
if block execution will not happen when the conditional statement returns `undefined or null or false or 0`
Hence we can ignore adding any special cases for v1 question

File: org.ekstep.questionunit-1.0/renderer/plugin.js (hunk 1 above)
Comment (sivashanmugam-funtoot) [reply org.ekstep.questionunit-1.0/renderer/plugin.js]: Done changes
https://github.com/project-sunbird/sunbird-content-plugins/pull/139/

File: org.ekstep.questionunit-1.0/renderer/plugin.js (hunk 2 above)
Comment (sivashanmugam-funtoot) [reply org.ekstep.questionunit-1.0/renderer/plugin.js]: Resolved as part of 
https://github.com/project-sunbird/sunbird-content-plugins/pull/139/

File: org.ekstep.questionunit.mcq-1.1/renderer/template/mcq-layouts.js (hunk 4 above)
Comment (sivashanmugam-funtoot) [reply org.ekstep.questionunit.mcq-1.1/renderer/template/mcq-layouts.js]: Fixed as part of 
https://github.com/project-sunbird/sunbird-content-plugins/pull/139/

**Pull request merged**